src/test/java/uk/gov/justice/digital/oasys/controller/AuthenticationControllerTest.java

```diff
@@ -48,22 +41,18 @@
     @MockBean
     private OasysAuthenticationRepository oasysAuthenticationRepository;
 
-    @Autowired
-    @Qualifier("globalObjectMapper")
-    private ObjectMapper objectMapper;
-
     @Value("${sample.token}")
     private String validOauthToken;
 
-    @Before
+    public static MockOAuthServer mockOAuthServer = new MockOAuthServer();
```

Comment:
`mockOAuthServer` is present in the `IntegrationTest` base class, is this instance needed?